File: totosuki/ABC/B/090.py
A, B = map(int, input().split())
cnt = 0

for n in range(A, B+1):
  n = list(str(n))
  n_rev = reversed(n)
  n = "".join(n)
  n_rev = "".join(n_rev)
  if int(n) == int(n_rev):
    cnt += 1

print(cnt)

File: OCH_Tao/ABC/B/090.js
const [A, B] = ((require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"))[0].split(" ")).map(x => parseInt(x, 10));
let cnt = 0;
for (let i = A; i < B + 1; i++) {
  let X = [...String(i)];
  if (String(i) === (X.reverse()).join("")) {
    cnt++;
  }
}
console.log(cnt);